site/src/nav/DownloadButton.css

```diff
@@ -26,3 +26,37 @@
     display: none;
   }
 }
+
+div.download-icon {
+  width:20px;
+  height:20px;
+  justify-content: center;
+  display: flex;
+}
+
+svg.icon-refresh {
+  top: 50%;
+  left: 50%;
+  width: 20px;
+  height: 20px;
+  animation: spin-animation 1s infinite ease;
+}
+
+svg.icon-refresh > circle.primary {
+  fill: white;
+}
+
+svg.icon-refresh > path.secondary {
+  fill: cornflowerblue;
```

Comment:
Ideally we would use a variable from infima here. Perhaps https://github.com/facebookincubator/infima/blob/main/packages/core/styles/common/variables.pcss#L31?